packages/shared-ui/src/flowHandler/flowHandler.ts

```diff
@@ -79,6 +75,10 @@ export class FlowHandler {
     this.#changeFlow();
   }
 
+  dispose() {
+    this.#corbadoApp.authService.abortOngoingPasskeyOperation();
```

Comment:
I think we should have a `this.#corbadoApp.dispose()` method which should black box the dispose functionality and will also be easily scalable in the future.

Reply:
Agreed => I created a dispose method on CorbadoApp (you could argue now, that this method should then call dispose methods off all its own services (projectService, apiService, authService), but that takes it a bit too far for this PR I think.